src/main/resources/config/defaults-storage-driver-kafka.yml

```diff
@@ -7,6 +7,7 @@ storage:
     create:
       key:
         enabled: false
+      headers: null
```

Comment:
https://stackoverflow.com/questions/33510094/syntax-for-empty-dictionary-in-yaml